pkg/apis/sources/v1alpha1/jobsource_defaults.go

```diff
@@ -22,5 +22,6 @@ import (
 
 // SetDefaults implements apis.Defaultable
 func (js *JobSource) SetDefaults(ctx context.Context) {
-	// TODO(spencer-p) output type should have a default
+	// TODO(spencer-p) Document this default
+	js.Spec.OutputFormat = OutputFormatBinary
```

Comment:
Should this check that it's not overwriting something first?

Reply:
It should check before setting. Set default if nil or empty...

Reply:
Can't be nil, will check for empty string